pkg/controller/nodes/task/taskexec_context.go

```diff
@@ -140,13 +145,18 @@ func (t *Handler) newTaskExecutionContext(ctx context.Context, nCtx handler.Node
 	}
 
 	resourceNamespacePrefix := pluginCore.ResourceNamespace(t.resourceManager.GetID()).CreateSubNamespace(pluginCore.ResourceNamespace(pluginID))
+	maxAttempts := uint32(0)
```

Comment:
I don't think this is valid. Attempt has to be greater than 0 right?

@katrogan @kumare3 Do we set default maxAttempts to 3 ?

Reply:
yeah it looks like we always set the min attempts to 1 + the user specified value: https://github.com/lyft/flytepropeller/blob/96c9ee9e2a9288101de846be8f49fce8c244ea2c/pkg/compiler/transformers/k8s/utils.go#L24